Cargo.lock

```diff
@@ -221,12 +221,12 @@ checksum = "cdb031dd78e28731d87d56cc8ffef4a8f36ca26c38fe2de700543e627f8a464a"
 
 [[package]]
 name = "backtrace"
```

Comment:
a stdlib type of crate

Reply:
The stdlib actually uses this crate for its backtracing implementation